racket/collects/racket/init.rkt

```diff
@@ -12,3 +12,7 @@
 (provide (all-from-out racket
                        racket/enter
                        racket/help))
+
+;; Set the default replt to XREPL
+(with-handlers ([exn:fail? void])
```

Comment:
Couldn't this use the test that I suggested: `(collection-file-path "main.rkt" "xrepl" #:fail (lambda _ #f))`?